src/utils.ts

```diff
@@ -471,3 +473,45 @@ export const contextUtils = <
     parserServices,
   };
 };
+
+const getArgumentExpression = (
+  x: TSESTree.CallExpressionArgument
+): O.Option<TSESTree.Expression> =>
+  // TODO: isExpression?
```

Comment:
Can you turn this `TODO` into a `NOTE` explaining why we need this?

Reply:
66cbbfcb8fc646310db9594625f0c6258cf23ee1